cfme/configure/settings.py

```diff
@@ -343,6 +344,10 @@ def step(self):
         tabs.select_tab("Default Filters")
 
 
+class DefaultBootstrapSwitch(View):
```

Comment:
I think it will be better to rename this class to `DefaultViewAll` and original `DefaultViewAll` to `DefaultViewStep` opr something like this.